platform/spec/services/syncs/create_service_spec.rb

```
@@ -0,0 +1,37 @@
+RSpec.describe Syncs::CreateService do
+  let(:connection) { create(:connection) }
+  let(:service) { described_class.new(connection.id) }
+  let(:mock_schema) do
+    {
+        stream1: { "name" => "stream1", "supported_sync_modes" => ["full_refresh", "incremental"] },
+        stream2: { "name" => "stream2", "supported_sync_modes" => ["full_refresh"] }
+    }
+  end
+
+  before do
+    allow_any_instance_of(Catalogs::FetchSchemaService).to receive(:call).and_return(mock_schema)
```

rubocop

[convention] Avoid stubbing using `allow_any_instance_of`.

(RSpec/AnyInstance)